source/config/profileUpgradeSteps.py

```diff
@@ -414,3 +414,58 @@ def upgradeConfigFrom_12_to_13(profile: ConfigObj) -> None:
 	log.debug(
 		f"Handled cldr value of {setting!r}. List is now: {profile['speech']['symbolDictionaries']}",
 	)
+
+
+def upgradeConfigFrom_13_to_14(profile: ConfigObj):
+	"""Set [audio][outputDevice] to the endpointID of [speech][outputDevice], and delete the latter."""
+	try:
+		friendlyName = profile["speech"]["outputDevice"]
+	except KeyError:
+		log.debug("Output device not present in config. Taking no action.")
+		return
+	if friendlyName == "default":
```

Comment:
Does it actually ever happen?

In my current `nvda.ini` (schema version = 13), I have:
```
[speech]
	outputDevice = Mappeur de Sons Microsoft
```

which is the French version of the default output device. I guess in English you would have "Microsoft Sound Mapper", not "default".

Reply:
This may happen if the user was on a previous alpha version and had changed their output device. Regardless, I don't think being cautious here is any harm.